tst/testbugfix/00098.tst

```diff
@@ -0,0 +1,3 @@
+# 2005/08/26 (Max)
+gap> IsOperation(MutableCopyMat);
+true
```

Comment:
So the tests I am musing about above (checking all documented words for being defined) could also check them to match what the documentation says: is it really an operation / property / attribute / function / ... ?